lms/static/js/views/fields.js

```diff
@@ -495,6 +495,203 @@
             }
         });
 
+        FieldViews.ImageFieldView = FieldViews.FieldView.extend({
+
+            fieldType: 'image',
+
+            templateSelector: '#field_image-tpl',
+            uploadButtonSelector: '.upload-button-input',
+
+            titleAdd: gettext("upload a photo"),
+            titleEdit: gettext("change photo"),
+            titleRemove: gettext("remove"),
```

Comment:
@srpearce Thanks for the review. These strings will be visible when you hover over image placeholder.